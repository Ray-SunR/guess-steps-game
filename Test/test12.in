game 30 4 stdin stdin
-13333333
-123123
0
1
2
5
8
8
8
8
8
game 30 4 test12A1.txt stdin
8
9
10
100
299292929
-23
123
game 30 4 stdin test12A1.txt
8
9
10
100
299292929
-23
123
quit
