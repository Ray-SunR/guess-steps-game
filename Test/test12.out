Total is 30, max deduction is 4
Player A's move
Total is 29
Player B's move
Total is 28
Player A's move
Total is 27
Player B's move
Total is 26
Player A's move
Total is 24
Player B's move
Total is 20
Player A's move
Total is 16
Player B's move
Total is 12
Player A's move
Total is 8
Player B's move
Total is 4
Player A's move
Total is 0
Player A wins
Score is
A 1
B 0
Total is 30, max deduction is 4
Player B's move
Total is 26
Player A's move
Total is 25
Player B's move
Total is 21
Player A's move
Total is 20
Player B's move
Total is 16
Player A's move
Total is 15
Player B's move
Total is 11
Player A's move
Total is 10
Player B's move
Total is 6
Player A's move
Total is 5
Player B's move
Total is 4
Player A's move
Total is 3
Player B's move
Total is -1
Player B wins
Score is
A 1
B 1
Total is 30, max deduction is 4
Player A's move
Total is 26
Player B's move
Total is 25
Player A's move
Total is 21
Player B's move
Total is 20
Player A's move
Total is 16
Player B's move
Total is 15
Player A's move
Total is 11
Player B's move
Total is 10
Player A's move
Total is 6
Player B's move
Total is 5
Player A's move
Total is 4
Player B's move
Total is 3
Player A's move
Total is -1
Player A wins
Score is
A 2
B 1
